acceptance-tests/src/test/scala/ch/renku/acceptancetests/model/projects.scala

```diff
@@ -66,17 +69,19 @@ object projects {
       val desc =
         maybeDescription.getOrElse(prefixParagraph("An automatically generated project for testing: ").generateOne)
       val readmeTitle = s"test ${now.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH-mm-ss"))}"
-      val template    = maybeTemplate.getOrElse("Renku/python-minimal")
-      ProjectDetails(title, visibility, desc, readmeTitle)
+      val template    = Template(maybeTemplate.getOrElse(Refined.unsafeApply("Renku/python-minimal")))
```

Comment:
I guess you don't need to wrap the `"Renku/python-minimal"` into `Refined.unsafeApply`. Simply add 
`import eu.timepit.refined.auto._`

Reply:
I thought it had to be simpler! :) There was some other refactoring we wanted to do so I'll try to remember to fix that as well.